SingularityUI/app/components/common/Navigation.jsx

```diff
@@ -35,6 +35,26 @@ function isActive(navbarPath, fragment) {
 // put into page wrapper, render children
 const Navigation = (props) => {
   const fragment = props.location.pathname.split('/')[1];
+  const navTitle = config.navTitleLinks
```

Comment:
Needed a null check for the config field itself somewhere, otherwise it would break if it wasn't set.